DevFest Web Development Track.md

```diff
@@ -131,7 +131,7 @@ This is a very basic directory structure for a Flask webapp.
 <a href="#top" class="top" id="hello-world-in-flask">Top</a>
 ## 1.2 Hello World in Flask
 
-Download the starter code for level 1 [here](http://dl.dropboxusercontent.com/s/z7lpnxld4ew0nfi/level0-done.zip?dl=0). Make sure that you move your `Vagrantfile` and `bootstrap.sh` into the same directory as the code. For the rest of the tutorial, we'll assume that you're inside of your Vagrant box and editing your code with Sublime Text.
+Download the starter code for level 1 [here](http://dl.dropboxusercontent.com/s/z7lpnxld4ew0nfi/level0-done.zip?dl=0). Make sure that you move your `bootstrap.sh` into the same directory as the code. For the rest of the tutorial, we'll assume that you're inside of your `virtualenv` virtual environment and editing your code with Sublime Text.
```

Comment:
1. I think bootstrap.sh is for Vagrant, since it definitely comes with the vagrant env setup zip.

2. Maybe reiterate the commands to enter virtualenv in just a large code block:

Rename the downloaded starter code to "reading-list-app"
```bash
cd reading-list-app
virtualenv venv
source venv/bin/activate
sudo pip install -r requirements.txt
```

3. And explain the last line: we have a requirements file in the starter code download that contains all the dependencies. install of these inside the venv with pip